prometheus/histogram.go

```diff
@@ -604,6 +611,13 @@ func (h *constHistogram) Write(out *dto.Metric) error {
 	return nil
 }
 
+func (h *constHistogram) GetExemplars() []*dto.Exemplar {
```

Comment:
This library has avoided adding exporter Getters to metric types so far. I would suggest to keep it that way for consistency. (The reason behind that is a different story. Should we ever change the approach, it should be done in an informed and consistent way in a separate PR, not snug in as a byproduct.)

Reply:
All good, I also raised this internally and it turns out we didn't need it anyway 👍